Sources/SmithyIdentity/CRTAdapters/AWSCredentialIdentityResolverCRTAdapter.swift

```diff
@@ -0,0 +1,21 @@
+//
```

Comment:
Things that interface with CRT for getting AWS credential identity have been moved from aws-sdk-swift to `SmithyIdentity` module here.